circleguard/comparer.py

```diff
@@ -44,6 +45,9 @@ def __init__(self, replays1, replays2=None):
 
         # filter beatmaps we had no data for
         self.replays1 = [replay for replay in replays1 if replay.replay_data is not None]
+        for replay in self.replays1:
+            if hasattr(replay, 'should_translate'):
```

Comment:
This is the hack I put in to enable the test. Not sure if there's a better way due to the lazy loading.